src/main/java/com/tstepnik/planner/repository/TaskRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.tstepnik.planner.repository;
+
+import com.tstepnik.planner.domain.Task;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long> {
+
+    @Query("SELECT t FROM Task t WHERE t.user.login=?1")
```

Comment:
Why are you adding custom query if it does exactly same thing just method name would do?

Reply:
didn't see method name find by user Id, is only find by user.

Reply:
Not looking good enough then :)

Reply:
:(